### DISCUSSION TITLE
IKEA motion detector and shortcut button not working any more

### DISCUSSION THREAD

**Discussion by soenkekueper** [Koenkk/zigbee2mqtt#10192] [General]
Hey,

since some time i've some trouble with my zigbee2mqtt and zzh-stick.
I've one IKEA motion detector (E1525/E1745) and one Shortcut Button (E1812).
To check the firmware, i've paired the shortcut button to ikea gateway, app says 2.3.080.
Both were running without any problems, i've recently recognized that both are not working
any more and try to figure out why.

Not working means: If i press the button - nothing happens.
I've removed the shortcut button several times and tried to add again, but all i can see from the logs is an timeout exception:

```
Zigbee2MQTT:error 2021-12-15 18:08:40: Failed to configure '0x8cf681fffe32e41b', attempt 1 (Error: Bind 0x8cf681fffe32e41b/1 genPowerCfg from '901' failed (AREQ - ZDO - bindRsp after 10000ms)
    at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7))
```

In the logs attached, i've started up and deleted the shortcut button two times. The second time i've pressed it some times after start paring,
seems to work "Succesfully configured" message is shown. But afterwards nothing is received from the button :/

I've updated to the latest docker image with zigbee2mqtt 1.22.1 and re-flashed the coordinator firmware CC2652R_coordinator_20210708 without any success :/

Do you have any idea?

Thanks a lot!
Log with herdsman logging:
[zigbee2mqtt.txt](https://github.com/Koenkk/zigbee2mqtt/files/7721476/zigbee2mqtt.txt)

Sönke

**Comment by martisak** [comment]
I am having similar issues with a remote. In the IKEA app I saw that they have pushed some new firmware that supports double click among other things. 

At startup it sends some information, but nothing on click.

`"{"action":null,"battery":null,"click":null,"device":{"applicationVersion":33,"dateCode":"20210722","friendlyName":"Mayas remote","hardwareVersion":1,"ieeeAddr":"0x0c4314fffe1c54c1","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","model":"E1743","networkAddress":28039,"powerSource":"Battery","softwareBuildID":"2.3.079","stackVersion":98,"type":"EndDevice","zclVersion":3},"last_seen":"2021-12-19T13:10:49.995Z","linkquality":207,"update_available":null}"`
